sdk/src/main/java/io/dapr/utils/Constants.java

```diff
@@ -60,6 +60,16 @@
    */
   public static final String ACTOR_TIMER_RELATIVE_URL_FORMAT = ACTORS_BASE_URL + "/%s/%s/timers/%s";
 
+  /**
+   * Environment variable name for dapr api token.
+   */
+  public static final String DAPR_API_TOKEN = "DAPR_API_TOKEN";
```

Comment:
Please, do not add constants here. I actually think this class should be gone and constants should be declared only when they are used. There might be a few shared constants here but it should be kept to a minimum.

Reply:
Is there a reason why we don't want to do this? This constant is being used in three files, so I find it reasonable to declare here.

Reply:
If constants are shared, it should be an owner with the right context. Having this wild card class with every constant in the project hurts readability. This was imported from the .Net class and needs refactoring. This is why I am avoiding adding more to it.